src/OctoshiftCLI.Tests/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandlerTests.cs

```diff
@@ -343,6 +343,58 @@ public async Task Happy_Path_Full_Flow_Bbs_Credentials_Via_Environment()
             ));
         }
 
+        [Fact]
+        public async Task Happy_Path_Uploads_To_Github_Storage()
+        {
+            // Arrange
+            _mockBbsApi.Setup(x => x.StartExport(BBS_PROJECT, BBS_REPO)).ReturnsAsync(BBS_EXPORT_ID);
+            _mockBbsApi.Setup(x => x.GetExport(BBS_EXPORT_ID)).ReturnsAsync(("COMPLETED", "The export is complete", 100));
+            _mockBbsArchiveDownloader.Setup(x => x.Download(BBS_EXPORT_ID, It.IsAny<string>())).ReturnsAsync(ARCHIVE_PATH);
+            _mockFileSystemProvider.Setup(x => x.ReadAllBytesAsync(ARCHIVE_PATH)).ReturnsAsync(ARCHIVE_DATA);
+            _mockGithubApi.Setup(x => x.GetOrganizationId(GITHUB_ORG).Result).Returns(GITHUB_ORG_ID);
+            _mockGithubApi.Setup(x => x.CreateBbsMigrationSource(GITHUB_ORG_ID).Result).Returns(MIGRATION_SOURCE_ID);
+            _mockGithubApi.Setup(x => x.UploadArchiveToGithubStorage(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<FileStream>()).Result).Returns("gei://archive/");
```

Comment:
This mock can be changed to match what we have for the gei counterpart

Reply:
no the test is fine, my comment is to make the much setup for `UploadArchiveToGithubStorage` such that it expects the exact input args exactly they way it was done for the GEI command handler test.

Reply:
Ahh I understand what you mean now

Reply:
Updated [here]([03b7266](https://github.com/github/gh-gei/pull/1280/commits/03b7266bfb81ca66de1bd65cb94e0f937ea1d33e))